src/addons/mod.rs

```diff
@@ -1,7 +1,8 @@
-//! # MultiTest add-ons
-//!
-//! Additional components and functionalities used to enhance
-//! or customize tests of CosmWasm smart contracts.
+
```

Comment:
This empty line is problematic. The Rust formatter will remove all empty lines before it encounters the first line  starting with `//!` - the file or module or crate level documentation.